README.rst

```diff
@@ -179,6 +186,62 @@ $ curl https://httpbin.org/get?show\_env=1
       "url": "http://httpbin.org/get?show_env=1"
     }
 
+$ curl -XPOST https://httpbin.org/file/dir1/dir2?name=myfile.json
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    {
+      "file_path": "/tmp/dir1/dir2/myfile.json"
```

Comment:
Agreed.  The user only needs to know of the relative path, we don't need to return the entire path.  I will make that update